### PR TITLE
СGE-172 Bug with empty messages on registering user with not valid credentials

### PR DESCRIPTION
**Pull request by drw85**
Checked all cases with invalid credentials for registering new user.
Added message on success user registering.
Covered with tests.

### CODE REVIEW

File: src/main/java/com/hillel/items_exchange/exception/handler/GlobalExceptionHandler.java [hunk 1]
```diff
@@ -67,7 +68,9 @@ public ErrorMessage handleRuntimeException(RuntimeException e, ServletWebRequest
     }
 
     private ErrorMessage logAndGetErrorMessage(ServletWebRequest request, Exception e, Level level) {
-
+        if (e.getClass().equals(UndeclaredThrowableException.class)){
```
Comment (rpkyrych) [inline src/main/java/com/hillel/items_exchange/exception/handler/GlobalExceptionHandler.java:71]: `if (e instanceof UndeclaredThrowableException){` looks like more elegant

File: src/main/java/com/hillel/items_exchange/exception/handler/GlobalExceptionHandler.java [hunk 2]
```diff
@@ -67,7 +68,9 @@ public ErrorMessage handleRuntimeException(RuntimeException e, ServletWebRequest
...
+            e = (Exception) ((UndeclaredThrowableException) e).getUndeclaredThrowable();
```
Comment (rpkyrych) [inline src/main/java/com/hillel/items_exchange/exception/handler/GlobalExceptionHandler.java:72]: Could you explain who throws such exception?

File: src/main/resources/messages_ru.properties [hunk 3]
```diff
@@ -32,6 +32,7 @@ subcategory.not-deletable=Подкатегория не может быть уд
 
 user.not-owner=Пользователь не является владельцем данного объявления: 
 user.not-found=Пользователь не найден, username/email: 
+user.created=Создан новый пользователь:{username}
```
Comment (rpkyrych) [inline src/main/resources/messages_ru.properties:35]: I observe extra spaces into English message text but they aren't here. Make both messages more consistent.

File: src/test/resources/messages_ru.properties [hunk 4]
```diff
@@ -32,6 +32,7 @@ subcategory.not-deletable=Подкатегория не может быть уд
 
 user.not-owner=Пользователь не является владельцем данного объявления: 
 user.not-found=Пользователь не найден, username/email: 
+user.created=Создан новый пользователь:{username}
```
Comment (rpkyrych) [inline src/test/resources/messages_ru.properties:35]: The same about the message here.

**Review by rpkyrych — CHANGES_REQUESTED** [review]
Small changes need to be provided.

File: src/main/java/com/hillel/items_exchange/exception/handler/GlobalExceptionHandler.java (hunk 1 above)
Comment (drw85) [reply src/main/java/com/hillel/items_exchange/exception/handler/GlobalExceptionHandler.java]: Agree

File: src/main/resources/messages_ru.properties (hunk 3 above)
Comment (drw85) [reply src/main/resources/messages_ru.properties]: Done

File: src/test/resources/messages_ru.properties (hunk 4 above)
Comment (drw85) [reply src/test/resources/messages_ru.properties]: Done

File: src/main/java/com/hillel/items_exchange/exception/handler/GlobalExceptionHandler.java (hunk 2 above)
Comment (drw85) [reply src/main/java/com/hillel/items_exchange/exception/handler/GlobalExceptionHandler.java:72]: It's 'validate' method in UserRegistrationDtoValidator thru @SneakyThrows annotation.

**Review by rpkyrych — APPROVED** [review]

**Review by svyatoslav145 — APPROVED** [review]

**Review by Jack11M — APPROVED** [review]

**Pull request merged**